include/nuttx/wireless/bluetooth/bt_core.h

```diff
@@ -217,4 +217,20 @@ static inline int bt_addr_le_to_str(FAR const bt_addr_le_t *addr, char *str,
  * Public Function Prototypes
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: bt_add_services
+ *
+ * Description:
+ *   Register services and start advertising.
+ *
+ * Input Parameters:
+ *   None
+ *
+ * Returned Value:
+ *   Zero in case of success or negative value in case of error.
+ *
+ ****************************************************************************/
+
+int bt_add_services(void);
```

Comment:
should we move the service to userspace?

Reply:
I would say no, because I don't see the value and don't have the time or interest right now, but I am not against someone submitting a PR that proves me wrong.

Reply:
In general I agree that services should be defined at application level, but currently I do not see infrastructure for that. Probably a proper `ioctl` service should be added to support it.